entrypoint.sh

```diff
@@ -42,7 +40,14 @@ fi
 
 ### GEOUPDATEIP
 echo "LAUNCH GEOIP UPDATE"
-/usr/bin/geoipupdate --verbose --output --database-directory="${GEOIPUPDATE_DB_DIR}"
+if [ "${GEOIPUPDATE_DRYRUN-default}" == "default" ]; then #if GEOIPUPDATE_DRYRUN is not set
+    /usr/bin/geoipupdate --verbose --output --database-directory="${GEOIPUPDATE_DB_DIR}"
+else
+    echo "DRY MODE ON" #if GEOIPUPDATE_DRYRUN is set
```

Comment:
```suggestion
    echo "DRY-RUN ON"
```

- Unneeded comment
- Keep the same wording (e.g. "Dry Run")